### PR TITLE
Fixed #94: Create CONTRIBUTING.adoc

### LINKED ISSUE #94: Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jiridanek**

### CODE REVIEW

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+The following things happen when using the "Squash and merge" button to commit your changes:
+
+1. The PR is automatically closed
+2. The github issue `#90` is closed
+3. The commit on the main branch is referenced in issue `#90`
+4. There is a hyperlink to the Github issue `#90` from the commit.
+
+For porting upstream qpid-dispatch issues to skupper-router you might use the standard `DISPATCH-XXXX: <Commit message>`  in your commit message but make sure you commit your change via the Github "Squash and merge" button.
+
+Please pull up any open PR on the skupper-router (for example - https://github.com/skupperproject/skupper-router/pull/80 ) and make sure you see the "Squash and merge" button.
```
Comment (ganeshmurthy) [inline CONTRIBUTING.adoc:27]: Can we just remove the link https://github.com/skupperproject/skupper-router/pull/80. Just say "Please pull up any open PR on the skupper-router and make sure you see the "Squash and merge" button.

**Review by ganeshmurthy — APPROVED** [review]

File: CONTRIBUTING.adoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
+Whenever you want to commit a change to the skupper-router repo, please first create a PR so it can be reviewed and approved by others.
```
Comment (ganeshmurthy) [inline CONTRIBUTING.adoc:1]: RAT check failure

File: CONTRIBUTING.adoc (hunk 1 above)
Comment (jiridanek) [reply CONTRIBUTING.adoc:27]: done

File: CONTRIBUTING.adoc (hunk 2 above)
Comment (jiridanek) [reply CONTRIBUTING.adoc:1]: added header

**Pull request merged**